scripts/sysdig-deploy/chart-version-bump.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+set -e
+
+na_chart_path="charts/node-analyzer"
```

Comment:
is it really needed to shorten node-analyzer to na here? i would prefer to avoid it if possible